spec/controllers/github_auth_controller_spec.rb

```diff
@@ -1,5 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe GithubAuthController, type: :controller do
+  describe 'GET #callback' do
+    let(:github_return_code) { 'github_code' }
+    subject { get :callback, params: { client_type: :member, code: github_return_code } }
+    let!(:token_result) { { access_token: 'token' } }
+    before do
+      expect(Octokit).to receive(:exchange_code_for_token)
+        .with(github_return_code, ENV['GH_AUTH_ID'], ENV['GH_AUTH_SECRET'])
+        .and_return(token_result)
+    end
 
+    it 'sets session values correctly' do
+      subject
+      expect(session[:access_token]).to eq(token_result[:access_token])
+      expect(session[:client_type]).to eq('member')
+    end
+
+    it { expect(subject).to redirect_to(root_path) }
+
+    context 'with settings as callback_action and test_org as gh_org' do
```

Comment:
no seria mejor aqui algo mas descriptivo?? esas dos cosas las puedo ver en el codigo mas abajo...

tal vez `context 'access to settings for an organization '` (algo asi, si eso es realmente el contexto)